tests/h/views/activity_test.py

```diff
@@ -921,8 +943,9 @@ def routes(pyramid_config):
 
 
 @pytest.fixture
-def search(patch):
+def search(patch, pyramid_config):
     search = patch('h.views.activity.SearchController.search')
+    pyramid_config.register_service(FakeAnnotationStatsService(), name='annotation_stats')
```

Comment:
This should really be in its own fixture, not attached to the `search` fixture :)